### ISSUE TITLE
Unable to extend selection accross different chat messages

### ISSUE THREAD

**Issue by resolritter** [Nheko-Reborn/nheko#716]
### Describe the bug

When you start drag the selection across different elements in e.g. the web browser, the selection is extended. On Nheko, however, the selection stops at the end of the current element, meaning it's not possible to extend the selection across them.

I recommend seeing the attached video for more clarity.

### To Reproduce

1. Start selecting some message
2. Try to expand the selection to another message

I recommend seeing the attached video for more clarity.

### Screenshots

https://user-images.githubusercontent.com/17429390/132537322-74531475-0f9a-450d-a558-0e375753327b.mp4

In the video above I try to extend the selection from one message to another by dragging with the mouse, but it doesn't work.

### Version

0.8.2-3f8bb19b

### Operating system

Linux

### Installation method

Local build

### Did you use profiles?

- [ ] Profiles used?

**Comment by deepbluev7** [comment]
Yes, this is a known limitation for now. Each message is a different widget and not part of one document (like it is in a browser). We want to fix that at some point, but you simply don't get that for free in native Applications.
